examples/rust/sysvar/src/processor.rs

```diff
@@ -43,3 +43,98 @@ pub fn process_instruction(
 
     Ok(())
 }
+
+
+//Code for deserializing the SlotHash sysvar while remaining within the BPF compute limit 
+
+//entrypoint!(process_instruction2);
+fn process_instruction2(
+    _program_id: &Pubkey,
+    accounts: &[AccountInfo],
+    _instruction_data: &[u8],
+) -> ProgramResult {
```

Comment:
What I meant was to have all this code happen after line 43, in the normal `process_instruction` function.  Written this way, the processor will never be triggered